src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -0,0 +1,35 @@
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.Solenoid;
+import frc.robot.Constants.HardwareConstants;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class ClimberSubsystem extends SubsystemBase{
+    
+    //Hardware
+    private final Solenoid climberSV = new Solenoid(HardwareConstants.pneumaticsModuleType, HardwareConstants.kClimberSolenoid_portNumber);
+
+    //Variables
+
+
+    public ClimberSubsystem(){
+    }
+
+    public void climberActivate(boolean state){
```

Comment:
```suggestion
    public void activateClimber(boolean state){
```

These are action-based in their wording.